packages/loader/container-loader/src/container.ts

```diff
@@ -1331,13 +1343,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         });
 
         // Track membership changes and update connection state accordingly
-        protocol.quorum.on("addMember", (clientId, details) => {
-            this.connectionStateHandler.receivedAddMemberEvent(clientId);
```

Comment:
I'd expect to see receivedAddMemberEvent & receivedRemoveMemberEvent to become private after that change